workflow-examples/src/main/java/com/redhat/parodos/examples/TibcoWorkFlowConfiguration.java

```diff
@@ -0,0 +1,34 @@
+// @formatter:off
```

Comment:
why is it necessary to disable the formatter for the whole file?

Reply:
so I don't need to execute it for every change. I don't need this formatter.

Reply:
@gciavarrini fixed

Reply:
It is still there. Can you double check?